eng/pipelines/fluent_java_mgmt_docs.yml

```diff
@@ -0,0 +1,44 @@
+trigger: none #this is not a CI, the pipeline will be triggered manually
+
+variables:
+- group: 'GitHub Release Variables'
```

Comment:
We generally try to avoid hooking up variables groups in the yaml files and instead do it in the web ui. Also it doesn't appear as though you end up using this because you end up exporting it below so I'd suggest removing this.

Reply:
i set it up since the variables setup here are "suipposed" to show up as env vars in the shell scripts below... seems that is not true.. :)

ill pull this out.